aptos-move/vm-genesis/src/lib.rs

```diff
@@ -1505,7 +1687,11 @@ pub fn test_mainnet_end_to_end() {
             delegator_addresses: vec![pbo_account31, pbo_account32, pbo_account33],
             delegator_stakes: vec![PBO_DELEGATOR_STAKE, PBO_DELEGATOR_STAKE, PBO_DELEGATOR_STAKE],
         },
-        principle_lockup_time: 100,
+        multisig_admin: multisig_account04,
+        unlock_schedule_numerators: vec![],
+        unlock_schedule_denominator: 0,
```

Comment:
Same here, empty numerators and `0` denominator would result in a failure.

Reply:
Same as above. This is an ignored test with placeholder values that needs fixing. It is tracked in https://github.com/Entropy-Foundation/aptos-core/issues/46.